go-guru.el

```diff
@@ -319,11 +319,11 @@ If BUFFER, return the number of characters in that buffer instead."
   "Go to the OFFSETth byte in the current line."
   (goto-char (byte-to-position (+ (position-bytes (point-at-bol)) (1- offset)))))
 
-(defun go-guru--goto-pos (posn)
+(defun go-guru--goto-pos (posn other-window)
```

Comment:
Is passing function better than passing flag value ? If its argument is function, we can pass other `find-file` function like `find-function-other-frame` etc.